### PR TITLE
fix(Onboarding): Give consistency to Create / Confirm password view font-size

### LINKED ISSUE #7447: Onboarding: create password view still has wrong font-size
Here's what the create password view looks like:

Screenshot from 2022-09-20 13-08-57

Here's what the password confirmation screen looks like:

Screenshot from 2022-09-20 13-09-11

Notice how on the first screen the text in the view is smaller and has a smaller line height as well.
This is only the case for this particular view in the onboarding which is why it stands out quite a bit.

### PR DESCRIPTION
**Pull request by noeliaSD**
Fixes #7447

### What does the PR do

- Updated `Create` and `Confirm` password views according to new design decisions taken to give consistency to the onboarding screens (related to font-size).
- Updated related squish test cases: Screenshots updated with new font-size.

### Affected areas

- Onboarding `Create` and `Confirm` screens.
- Also can affect `Change Password` view since they share qml component.

### Screenshot of functionality 

https://user-images.githubusercontent.com/97019400/195112031-5db00385-e902-433b-bcfe-9620b6e20cfd.mov

### CODE REVIEW

**Comment by noeliaSD** [comment]
@PascalPrecht, could you  be the tester here? Thanks

File: test/ui-test/testSuites/suite_onboarding/shared/verificationPoints/VP-PWStrength-lower_very_weak [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
...88zmXd6IAAAAAYAvEGDd9vGWWhZsrnc7G9oPMXSFNBvsLm+yNAAAAAFsk3WyjIU1Dqdi1TEGpmOgyAAAAwLdWmmz+EIdi9xoNXbw1AAAA8JfFGDedaAghVMpJudSFB/5yKamUNRoAAADg2+tWEg0hhHKpCwMMXbkpAAAAsHG3+OheKW91uGDr7wgAAABsSpIktzI6sa5aTrfspMlCIamWxRkAAADg2+4WRydCCMViUi3f8qs3IUlCtZzYBhIAAAC+/W490RBCqFbSnsodDxr0VNLqnb8LAAAAcPtuPdGwrqd6Z7sA1UraU9VlAAAAgLtDcjuJhi/U6lmtkeWx0h8kSejRZQAAAIC7R4yxmGVZmt7uw3xPNU2SUG/GTiefZkOhkFTLiYkJAAAAuLvkk2hY127HejNrNG93wUo5qZZTuz8CAADA3SXGmGejYV2jGZutrNm6lWXLpaRcSitlLQYAAAC4+2RZlnQ6ndsfnfi6RjO227HVjp3smzsOhTQpFZNiMdFiAAAAgLta/omGL8uy0O7ETid2stjphCx+frckSdIkFAqhkCaFQlIsJHeg1wEAAABstTvbaAAAAADuHTHGVKMBAAAAyIuJBQAAACAfSZIYnQAAAAByk8YYsyzrdhkAAADA3W09ypA2m80kcagkAAAAcLtqtVq6vLys0QAAAADcpiRJ1tbW0uXl5W5XAgAAANz1Yow3b95MFxYW7NEAAAAA3KYkSebn59MbN26kqUMuAQAAgFu3HmJYXl5Osyybm5vrdj0AAADAXSxN07m5uRhjOjAw8Pbbb3e7HgAAAODu9vbbbw8ODqb9/f09PT2vv/76+nGXAAAAAJt17NixcrlcLpfTQqEwODi4vLz8xhtvZFmm3QAAAABsXIzx97///Y0bN0ZHR6vVavLee+9lWdbpdK5fv95qtR555JHJycksy5IkSZKk29UCAAAA3xbr6YQkSWKM602D2dnZd999N03TkZGRcrlcKBTSQqFQLBaLxeLw8HBfX9+JEyeef/75Y8eOffTRR0tLS2tra+urfJF0iDGu7yT55exDjHH9n1+clPknkxFfXPb1K61vfetb3/rWt771rW9961vf+ta3/rd8/SRJ1tbWlpaWzp079/rrr//iF784ceJEX1/f2NhYtVpdP9Ty/wOdlMOZRm8zUgAAAABJRU5ErkJggg==<Mask/>
```
Comment (caybro) [inline test/ui-test/testSuites/suite_onboarding/shared/verificationPoints/VP-PWStrength-lower_very_weak:4]: @iurimatias are we really putting binary blobs to Git? What about LFS?

File: ui/imports/shared/views/PasswordView.qml [hunk 2]
```diff
@@ -132,37 +133,30 @@ Column {
...
-        }
+    StatusBaseText {
+        id: introTxtField
+        Layout.fillWidth: true
+        text: "%1 <font color=\"%2\">%3</font>".arg(root.introText).arg(Theme.palette.dangerColor1).arg(root.recoverText)
+        font.pixelSize: root.highSizeIntro ? 15 : 12
+        color: Theme.palette.baseColor1
+        wrapMode: Text.WordWrap
+        horizontalAlignment: TextEdit.AlignHCenter
+        clip: true
```
Comment (caybro) [inline ui/imports/shared/views/PasswordView.qml:159]: Why the clip?

File: ui/imports/shared/views/PasswordView.qml [hunk 3]
```diff
@@ -132,37 +133,30 @@ Column {
         function isTooShort() { return newPswInput.text.length < root.minPswLen }
     }
 
-    spacing: 3 * Style.current.padding / 2
+    spacing: Style.current.bigPadding
     z: root.zFront
-    width: 416
+    clip: true
```
Comment (caybro) [inline ui/imports/shared/views/PasswordView.qml:138]: Why the clip? It generally indicates a workaround for some hidden problem :)

File: ui/app/AppLayouts/Onboarding/views/ConfirmPasswordView.qml [hunk 4]
```diff
@@ -49,104 +49,91 @@ Item {
...
-            StatusBaseText {
-                id: errorTxt
-                anchors.horizontalCenter: parent.horizontalCenter
-                font.pixelSize: 12
-                color: Theme.palette.dangerColor1
-                onTextChanged: {
-                    if(text === "") filler.visible = true
-                    else filler.visible = false
-                }
+        // TODO replace with StatusInput as soon as it supports password
```
Comment (caybro) [inline ui/app/AppLayouts/Onboarding/views/ConfirmPasswordView.qml:93]: We now have `StatusPasswordInput` :)

**Review by caybro — CHANGES_REQUESTED** [review]
Just some minor things, otherwise looks good!

**Review by PascalPrecht — APPROVED** [review]
Tested it and it looks much better IMO

**Comment by PascalPrecht** [comment]
One thing though... I think here the text is too close together isn't it? I think that part can be left untouched?

Screenshot from 2022-10-13 09-14-19

File: test/ui-test/testSuites/suite_onboarding/shared/verificationPoints/VP-PWStrength-lower_very_weak (hunk 1 above)
Comment (caybro) [reply test/ui-test/testSuites/suite_onboarding/shared/verificationPoints/VP-PWStrength-lower_very_weak:4]: Thanks :)

File: ui/imports/shared/views/PasswordView.qml [hunk 5]
```diff
@@ -132,54 +133,44 @@ Column {
...
+        id: introTxtField
+        Layout.fillWidth: true
+        text: "%1 <font color=\"%2\">%3</font>".arg(root.introText).arg(Theme.palette.dangerColor1).arg(root.recoverText)
+        font.pixelSize: root.highSizeIntro ? 15 : 12
+        color: Theme.palette.baseColor1
+        wrapMode: Text.WordWrap
+        horizontalAlignment: TextEdit.AlignHCenter
     }
 
     // TODO replace with StatusInput as soon as it supports password
```
Comment (caybro) [inline ui/imports/shared/views/PasswordView.qml:160]: Thanks for the fixes! <3 Now you can actually drop the TODO :)

**Comment by noeliaSD** [comment]
> One thing though... I think here the text is too close together isn't it? I think that part can be left untouched?
> 
> Screenshot from 2022-10-13 09-14-19

This is the final design implemented for this screen (I'm updating yours, @PascalPrecht, bc I realized the password input width was not correct there):

**Review by caybro — APPROVED** [review]
Nice work!

File: ui/imports/shared/views/PasswordView.qml (hunk 5 above)
Comment (noeliaSD) [reply ui/imports/shared/views/PasswordView.qml]: Oh.. leftover! Done!

**Comment by noeliaSD** [comment]
Pushed new commit, fixing `change_password` `e2e-test case` failure.

**Comment by benjthayer** [comment]
Hi @noeliaSD @PascalPrecht both the create password and confirm password screen copy should use the same pt size and line height - in the designs that is 15 Body which is 15pt with 22pt line height in regular weight (or whatever that font token maps to in the code). Hope that helps!

**Comment by noeliaSD** [comment]
Added tag `mayfail` in specific scenario from `suite_communities` that is broken / not reliable (not from changes done in this pr), and created task for addressing it: https://github.com/status-im/status-desktop/issues/7889

**Pull request merged**